docs/Contributing/Audit-logs.md

```diff
@@ -1216,6 +1216,29 @@ This activity contains the following fields:
 }
 ```
 
+## edited_software
+
+Generated when a software installer is updated in Fleet.
+
+This activity contains the following fields:
+- "software_title": Name of the software.
+- "software_package": Filename of the installer. `null` if the installer package was not modified.
```

Comment:
@iansltx updated this description in https://github.com/fleetdm/fleet/pull/22177/files earlier today, so I think to avoid conflicts it should now be:
```suggestion
- "software_package": Filename of the installer as of this update (including if unchanged).
```